<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addPackage</class>
 <widget class="QDialog" name="addPackage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="packagelabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Package 추가</string>
   </property>
  </widget>
  <widget class="QWidget" name="packagename" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>196</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="name_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="name_button">
       <property name="text">
        <string>확인</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="location" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QComboBox" name="loc_combo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loc_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="tag" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>196</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLineEdit" name="tag_line"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="tag_button_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="price" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>196</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLineEdit" name="price_line"/>
     </item>
     <item>
      <widget class="QPushButton" name="price_button_2">
       <property name="text">
        <string>확인</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="via" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="via_button_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QComboBox" name="via_combox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="trav_start_date" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="date_button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>111</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="PID">
      <property name="text">
       <string>PID :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="PID_LCD">
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pname_button">
      <property name="text">
       <string>패키지 이름</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="location_button">
      <property name="text">
       <string>지역</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="tag_button">
      <property name="text">
       <string>태그</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="price_button">
      <property name="text">
       <string>가격(원화)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="via_button">
      <property name="text">
       <string>경유</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="trav_date_button">
      <property name="text">
       <string>여행 출발일</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="trav_hour_button">
      <property name="text">
       <string>여행 출발 시간</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="how_long_button_2">
      <property name="text">
       <string>여행기간</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="trav_free_button">
      <property name="text">
       <string>자유여행</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="minppl_button">
      <property name="text">
       <string>최소인원</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="maxppl_button">
      <property name="text">
       <string>최대인원</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="trav_start_hour" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>340</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
    <property name="showGroupSeparator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="currentSection">
     <enum>QDateTimeEdit::HourSection</enum>
    </property>
    <property name="displayFormat">
     <string>h:mm</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="free_trav" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QPushButton" name="trav_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QComboBox" name="trav_combo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>2</number>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="how_long_trav" native="true">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="how_long_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="how_long_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="minppl" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="min_spin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="min_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="maxppl" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>490</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <widget class="QSpinBox" name="max_spinbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="max_button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="finish">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>510</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>location_button</sender>
   <signal>clicked()</signal>
   <receiver>location</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loc_button</sender>
   <signal>clicked()</signal>
   <receiver>location</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tag_button</sender>
   <signal>clicked()</signal>
   <receiver>tag</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tag_button_2</sender>
   <signal>clicked()</signal>
   <receiver>tag</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>473</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>price_button</sender>
   <signal>clicked()</signal>
   <receiver>price</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>298</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>price_button_2</sender>
   <signal>clicked()</signal>
   <receiver>price</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>via_button</sender>
   <signal>clicked()</signal>
   <receiver>via</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>via_button_2</sender>
   <signal>clicked()</signal>
   <receiver>via</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>406</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trav_date_button</sender>
   <signal>clicked()</signal>
   <receiver>trav_start_date</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trav_hour_button</sender>
   <signal>clicked()</signal>
   <receiver>trav_start_hour</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>start_button</sender>
   <signal>clicked()</signal>
   <receiver>trav_start_hour</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>450</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trav_free_button</sender>
   <signal>clicked()</signal>
   <receiver>free_trav</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>475</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>trav_button</sender>
   <signal>clicked()</signal>
   <receiver>free_trav</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>how_long_button_2</sender>
   <signal>clicked()</signal>
   <receiver>how_long_trav</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>how_long_button</sender>
   <signal>clicked()</signal>
   <receiver>how_long_trav</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minppl_button</sender>
   <signal>clicked()</signal>
   <receiver>minppl</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>419</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_button</sender>
   <signal>clicked()</signal>
   <receiver>minppl</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxppl_button</sender>
   <signal>clicked()</signal>
   <receiver>maxppl</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>max_button</sender>
   <signal>clicked()</signal>
   <receiver>maxppl</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>date_button</sender>
   <signal>clicked()</signal>
   <receiver>trav_start_date</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>340</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pname_button</sender>
   <signal>clicked()</signal>
   <receiver>packagename</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>name_button</sender>
   <signal>clicked()</signal>
   <receiver>packagename</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>via_button_2</sender>
   <signal>clicked()</signal>
   <receiver>via</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finish</sender>
   <signal>clicked()</signal>
   <receiver>addPackage</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>563</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
